go.mod

```diff
@@ -18,11 +18,12 @@ require (
 	github.com/mattn/go-isatty v0.0.3
 	github.com/mattn/go-sqlite3 v1.14.9
 	github.com/mattn/go-tty v0.0.0-20180219170247-931426f7535a
+	github.com/otherview/filerotatewriter v0.0.0-20240509103702-a305c09fd53a
```

Comment:
Do we need a lib for this? Would personally prefer to have less dependencies

Reply:
Yeah I know you mean, I didn't find a place that would make sense, so decided to use it as an external lib. We do need a package to wrap the rotation logic.
Where do you think it should be located ?

Reply:
Possibly `./api/logging` ?

Reply:
Actually thats not nice either, because then its under a `logging` package name.

Reply:
Yeah I was resistant to add that there because it would "ruin" the nice packaging of the `api` folder. What about `api/utils/logfile` ?

Reply:
Makes sense to me

Reply:
Done 👍